lib/plugin.js

```diff
@@ -48,7 +49,11 @@ middleware.auth = context => {
   const isAuthenticated = checkAuthenticatedRoute(context)
   const { accessToken = null } = context.store.state[moduleName]
 
-  if (!isAuthenticated || !!accessToken) return
+  if (!isAuthenticated || !!accessToken) {
+    errorLog('Not authenticate but access token exists.')
```

Comment:
you can't tell if a token exists from this lazy evaluation can you?

Reply:
yeah but i think we can just say "not authenticated"?

Reply:
Are we adding `nuxt-oauth - ` prefix to all the logs?